src/neo-vm/Types/Array.cs

```diff
@@ -58,7 +67,7 @@ public override bool Equals(StackItem other)
 
         public override bool GetBoolean()
         {
-            return _array.Count > 0;
+            return true;
```

Comment:
This is a breaking change, was this intended?

Reply:
It has never been used in the past. And this change is consistent with the behavior of TypeScript/JavaScript.

```js
var a = [];
if (a) {
  //should be executed here
}
```

Reply:
Sounds good to me.